research/flamby/fed_heart_disease/moon/config.yaml

```diff
@@ -6,3 +6,4 @@ n_clients: 4 # The number of clients in the FL experiment
 local_epochs: 1 # The number of epochs to complete for client (NOT USED FOR FLAMBY)
 batch_size: 64 # The batch size for client training (NOT USED FOR FLAMBY)
 local_steps: 100 # The number of local training steps to perform.
+federated_checkpointing: True # Indicates whether intermediate or latest checkpointing is used on the server side
```

Comment:
I think this is worth adding to the `fed_ixi/moon/config.yaml`. Because of the default for the config.get function, it doesn't change the functionality for `federated_checkpointing` but it makes it obvious how to turn it off.